mapper/tedge_mapper/src/c8y_mapper.rs

```diff
@@ -0,0 +1,39 @@
+use crate::error::*;
+use crate::mapper::*;
+use mqtt_client::Topic;
+
+#[derive(Debug)]
+pub struct CumulocityMapperConfig {
+    pub in_topic: Topic,
+    pub out_topic: Topic,
+    pub errors_topic: Topic,
+}
+
+impl Default for CumulocityMapperConfig {
+    fn default() -> Self {
+        Self {
+            in_topic: Topic::new("tedge/measurements").unwrap(),
+            out_topic: Topic::new("c8y/measurement/measurements/create").unwrap(),
+            errors_topic: Topic::new("tedge/errors").unwrap(),
+        }
+    }
+}
+
+impl Into<MapperConfig> for CumulocityMapperConfig {
+    fn into(self) -> MapperConfig {
+        MapperConfig {
+            in_topic: self.in_topic,
+            out_topic: self.out_topic,
+            errors_topic: self.errors_topic,
+        }
+    }
+}
```

Comment:
Prefer `From` over `Into`:
```suggestion
impl From<CumulocityMapperConfig> for MapperConfig {
    fn from(val: CumulocityMapperConfig) -> Self {
        MapperConfig {
            in_topic: val.in_topic,
            out_topic: val.out_topic,
            errors_topic: val.errors_topic,
        }
    }
}
```